cves/kernel/CVE-2019-9162.yml

```diff
@@ -167,8 +169,11 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: |
+    It is possible for a static analysis tool to discover this
+    vulnerability as it is extremely similar to an example of buffer overflow.
+    An array index error occurs and can read/write extra bytes in memory.
```

Comment:
What can read/write extra bytes? This sentence could be reworded.

Reply:
Updated